src/v2/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -619,10 +622,13 @@ describe("Shipping", () => {
 
       it("opens modal with current address values", async () => {
         expect(page.find(ShippingRoute).state().editAddressIndex).toBe(-1)
-        await page.find(`[data-test="edit-address"]`).first().simulate("click")
+        await page
+          .find(`[data-test="editAddressInShipping"]`)
+          .first()
+          .simulate("click")
         expect(page.find(ShippingRoute).state().editAddressIndex).toBe(0)
 
-        expect(page.find("AddressModal").length).toBe(2)
+        expect(page.find("AddressModal").length).toBe(1)
```

Comment:
😅  I'm not sure why we were previously expecting 2 of these modals, but this assertion makes me feel much more comfortable now. 👍